src/redux/actions/authActions.js

```diff
@@ -0,0 +1,143 @@
+import fetch from 'cross-fetch';
+
+export const SIGNUP = 'SIGN_UP';
+export const SIGNUP_REQUEST = 'REQUEST_SIGNUP';
+export const SIGNUP_FAILURE = 'SIGNUP_FAILURE';
+export const SIGNUP_SUCCESS = 'SIGNUP_SUCCESS';
+export const LOGOUT = 'LOGOUT';
+export const LOGIN = 'LOGIN';
+export const LOGIN_REQUEST = 'REQUEST_LOGIN';
```

Comment:
For consistency, we could do 'LOGIN_REQUEST' and 'SIGNUP_REQUEST', since you already did 'LOGOUT REQUEST'

Reply:
Oh, whoops. This Action was meant to be titled 'LOGIN_REQUEST'. Will fix it now. Thanks for catching that.